### PR TITLE
EUI-8453 QM Raise a query error scenarios

### PR DESCRIPTION
**Pull request by johnbenjamin-hmcts**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-8453

### Change description ###
Query Management - Raise a query (Error)

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[X] No
```

### CODE REVIEW

File: src/cases/containers/query-management-container/query-management-container.component.html [hunk 1]
```diff
@@ -1,7 +1,21 @@
 <exui-app-header></exui-app-header>
-
 <div class="govuk-width-container">
   <main class="govuk-main-wrapper" id="main-content" role="main">
+    <!-- Error message summary -->
+    <div *ngIf="errorMessages.length > 0" class="govuk-error-summary" aria-labelledby="error-summary-title" role="alert" tabindex="-1" data-module="govuk-error-summary">
+      <h2 class="govuk-error-summary__title" id="error-summary-title">
+        {{'There is a problem' | rpxTranslate}}
```
Comment (Daniel-Lam) [inline src/cases/containers/query-management-container/query-management-container.component.html:7]: @johnbenjamin-hmcts - possible linting failure; the pipeline linter may ask for a space after `{{` and before `}}`

File: src/cases/containers/query-management-container/query-management-container.component.html [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+      </h2>
+      <div *ngFor="let errorMessage of errorMessages" class="govuk-error-summary__body">
+        <ul class="govuk-list govuk-error-summary__list">
+          <li>
+            <a [id]="'error-' + errorMessage.fieldId"
+              (click)="navigateToErrorElement(errorMessage.fieldId)"
+              class="validation-error">{{errorMessage.description | rpxTranslate}}</a>
```
Comment (Daniel-Lam) [inline src/cases/containers/query-management-container/query-management-container.component.html:14]: @johnbenjamin-hmcts - possible linting failure; the pipeline linter may ask for a space after `{{` and before `}}`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - very minor observations on linting; otherwise OK

File: src/cases/containers/query-management-container/query-management-container.component.html (hunk 1 above)
Comment (johnbenjamin-hmcts) [reply src/cases/containers/query-management-container/query-management-container.component.html]: @Daniel-Lam - It was a copy-paste from Toolkit to MC. I have now updated the code.

File: src/cases/containers/query-management-container/query-management-container.component.html (hunk 2 above)
Comment (johnbenjamin-hmcts) [reply src/cases/containers/query-management-container/query-management-container.component.html]: @Daniel-Lam - It was a copy-paste from Toolkit to MC. I have now updated the code.

File: src/cases/containers/query-management-container/query-management-container.component.ts [hunk 3]
```diff
@@ -26,7 +26,7 @@ export class QueryManagementContainerComponent implements OnInit {
       body: new FormControl(null, Validators.required),
       isHearingRelated: new FormControl(null, Validators.required),
       hearingDate: new FormControl(null),
-      attachments: new FormControl([])
+      attachments: new FormControl([] as Document[])
```
Comment (Daniel-Lam) [inline src/cases/containers/query-management-container/query-management-container.component.ts:29]: @johnbenjamin-hmcts - can you check with @DavidJayakumar whether this is the correct type: there is `Document` and also `FormDocument`

**Review by Daniel-Lam — CHANGES_REQUESTED** [review]
@johnbenjamin-hmcts - one minor thing to check

File: src/cases/containers/query-management-container/query-management-container.component.ts (hunk 3 above)
Comment (johnbenjamin-hmcts) [reply src/cases/containers/query-management-container/query-management-container.component.ts:30]: @Daniel-Lam I checked the base branch _feature/query-management_ and it is Document and not FormDocument. Might be that the changes are not yet merged to the base branch yet.

**Review by Daniel-Lam — APPROVED** [review]

**Pull request merged**